org.mixedrealitytoolkit.input/Tracking/TrackedPoseDriverExtensions.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+using UnityEngine.InputSystem.XR;
+using UnityEngine.XR;
+
+namespace MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Extensions for Unity's <see cref="TrackedPoseDriver"/>
+    /// </summary>
+    public static class TrackedPoseDriverExtensions
+    {
+        /// <summary>
+        /// Gets the tracking state of the <see cref="TrackedPoseDriver"/>. If the tracking state is not available, returns false.
+        /// </summary>
+        public static bool TryGetTrackingState(this TrackedPoseDriver driver, out InputTrackingState state)
+        {
+            state = InputTrackingState.None;
+            var trackingStateAction = driver.trackingStateInput.action;
+            if (trackingStateAction == null || trackingStateAction.bindings.Count == 0)
+            {
+                return false;
+            }
+
+            if (trackingStateAction.enabled)
+            {
+                return false;
+            }
+
+            if (!trackingStateAction.HasAnyControls())
+            {
+                return false;
+            }
+
+            state = (InputTrackingState)trackingStateAction.ReadValue<int>();
+            return true;
+        }
+
+        /// <summary>
+        /// Gets the tracking state of the <see cref="TrackedPoseDriver"/>.
+        /// </summary>
+        /// <remarks>
+        /// If the <see cref="TrackedPoseDriver"/> has no tracking state action or the action has no bindings, it will return `<see cref="InputTrackingState.Position"/> |
+        /// <see cref="InputTrackingState.Rotation"/>`. If the action is enabled, it will return `<see cref="InputTrackingState.None"/>`. If the action has controls, it will return the value of the action.
```

Comment:
```suggestion
/// <see cref="InputTrackingState.Rotation"/>`. If the action is disabled, it will return `<see cref="InputTrackingState.None"/>`. If the action has controls, it will return the value of the action.
```